### PR TITLE
CB-5145 - metrics for inter-service REST calls

### PR DESCRIPTION
**Pull request by attilapalfi92**

### CODE REVIEW

File: common/src/main/java/com/sequenceiq/cloudbreak/jerseyclient/AugmentedRestClient.java [hunk 1]
```diff
@@ -7,5 +7,5 @@
 
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
-public @interface RetryingRestClient {
+public @interface AugmentedRestClient {
```
Comment (topolyai5) [inline common/src/main/java/com/sequenceiq/cloudbreak/jerseyclient/AugmentedRestClient.java]: What was the problem with RetryingRestClient? This is retrying only like previously, right?

Comment (bbihari) [reply common/src/main/java/com/sequenceiq/cloudbreak/jerseyclient/AugmentedRestClient.java]: 'augmented' is a bit too general

Comment (attilapalfi92) [reply common/src/main/java/com/sequenceiq/cloudbreak/jerseyclient/AugmentedRestClient.java]: @topolyai5, no, please check the aspect's code.

Comment (attilapalfi92) [reply common/src/main/java/com/sequenceiq/cloudbreak/jerseyclient/AugmentedRestClient.java]: @bbihari any suggestions?

Comment (attilapalfi92) [reply common/src/main/java/com/sequenceiq/cloudbreak/jerseyclient/AugmentedRestClient.java]: MetricCollectingAndRetryingRestClient would be better? I think the current name is fine.

**Pull request merged**